src/app/modules/group/pages/group-by-id/group-by-id.component.ts

```diff
@@ -49,7 +49,7 @@ export class GroupByIdComponent implements OnDestroy {
           route: groupRoute(group.id, route.path),
           breadcrumbs: {
             category: GROUP_BREADCRUMB_CAT,
-            path: [{ title: group.name, navigateTo: this.router.createUrlTree(urlArrayForGroupRoute(route, 'details')) }],
+            path: [{ title: group.name, navigateTo: this.router.createUrlTree(urlArrayForGroup(route, 'details')) }],
```

Comment:
As it is done for items, it 's better to pass a block to the breadcrumb for navigation so that the link is the route to the same subpage as the current one.
But that could be done in a future PR.